init.lua

```diff
@@ -1,16 +1,24 @@
-vim.cmd [[
-  set packpath-=~/.config/nvim
-  set packpath-=~/.config/nvim/after
-  set packpath-=~/.local/share/nvim/site
-  set packpath^=~/.local/share/lunarvim/site
-  set packpath^=~/.config/lvim
+--    __                          _    ___
+--   / /   __  ______  ____ _____| |  / (_)___ ___
+--  / /   / / / / __ \/ __ `/ ___/ | / / / __ `__ \
+-- / /___/ /_/ / / / / /_/ / /   | |/ / / / / / / /
+--/_____/\__,_/_/ /_/\__,_/_/    |___/_/_/ /_/ /_/
 
-  set runtimepath-=~/.config/nvim
-  set runtimepath-=~/.config/nvim/after
-  set runtimepath+=~/.config/lvim
-  set runtimepath^=~/.local/share/lunarvim/lvim/after
-]]
--- vim.opt.rtp:append() instead of vim.cmd ?
+--- Bootstrap
+vim.opt.rtp:append(os.getenv "HOME" .. "/.local/share/lunarvim/lvim")
```

Comment:
local home = os.getenv "HOME"

Reply:
or
```
local home = vim.loop.os_homedir()
```
what shall simplify future modifications for MS Windows environment.

Anyway, it seems usable as is with the repeated `os.getenv`